gen/dcos-config.yaml

```diff
@@ -473,8 +473,6 @@ package:
       {
         "PATH": "/usr/bin:/bin",
         "SHELL": "/usr/bin/bash",
-        "LD_LIBRARY_PATH": "/opt/mesosphere/lib",
```

Comment:
Cody tried to remove this in this PR: https://github.com/dcos/dcos/pull/707/

That PR isn't merged because the SDK frameworks weren't ready at that time. Are they ready now? =)